content/blog/2018-06-05-unconf18_recap_1.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: "unconf_recap_1"
+title: "Unconf18 projects 1: mchtoolbox, revtools, dataspice, rOpenSciEd, rOpenInterviews"
+date: 2018-06-05
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+categories: blog
+topicid: 
+tags:
+- community
+- ropensci
+- meetings
+- unconf
```

Comment:
add tag
`- unconf18`

cause I think it would be helpful to be able to refer to posts from individual unconfs, ozunconf etc